### PR TITLE
Dobavil tablicy s pomosch'u grid

### PR DESCRIPTION
**Pull request by Green1ce**
Mo9 versi9

### CODE REVIEW

File: Sergey/Medium/index.html [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+                    </button>
+                </a>
+            </div>
+        </div>
+    </div>
+    <div class="article">
+        <div class="article_container">
+            <div class="article_container_left">
+                <h1 class="article_header">Where good ideas find you</h1>
+                <p class="article_text">Read and share new perspectives on just about any topic. Everyone's welcome. <a href="https://about.medium.com/">Learn more.</a></p>
```
Comment (iruseg00) [inline Sergey/Medium/index.html]: Not readable.

File: Sergey/Medium/index.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+            <div class="items_header">
+                <i class="fas fa-arrow-alt-circle-up"></i>
+                <span>Trending on medium</span>
+            </div>
+        </div>
+        <div class="items_container2">
+            <div class="items_items">
+                <div class="items_number">
+                    <div class="number">
+                        <span>01</span>
```
Comment (iruseg00) [inline Sergey/Medium/index.html]: Why is span here?

File: Sergey/Medium/index.html [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+                    </div>
+                    <div class="item">
+                        <div class="fuck">
```
Comment (iruseg00) [inline Sergey/Medium/index.html]: The class name should describe the essence of the element.

File: Sergey/Medium/index.html [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+                            <i class="far fa-circle"></i>
+                            <h4>Katie Couric in Wake-Up Call</h4>
+                        </div>
+                        <h2>In Your Business With Dr. B</h2>
+                        <div class="date">
+                            <span>Oct 21</span>
+                            <span> | </span>
+                            <span>7 min read</span>
+                        </div>
```
Comment (iruseg00) [inline Sergey/Medium/index.html:63]: Can we remove the 3 span and leave the text in the one div?

File: Sergey/Medium/index.html [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+                    </div>
+                </div>
+                <div class="items_number">
+                    <div class="number">
+                        <span>02</span>
```
Comment (iruseg00) [inline Sergey/Medium/index.html]: Similarly.

File: Sergey/Medium/index.html [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+                        <div class="fuck">
+                            <i class="far fa-circle"></i>
+                            <h4>Tressie McMillan Cottom</h4>
+                        </div>
+                        <h2>One Thing About Being A MacArthur Fellow Pisses Me Off</h2>
+                        <div class="date">
+                            <span>Oct 21</span>
+                            <span> | </span>
+                            <span>3 min read</span>
+                        </div>
```
Comment (iruseg00) [inline Sergey/Medium/index.html:78]: Similarly.

File: Sergey/Medium/index.html [hunk 7]
```diff
@@ -0,0 +1,188 @@
...
+                    </div>
+                </div>
+                <div class="items_number">
+                    <div class="number">
+                        <span>03</span>
```
Comment (iruseg00) [inline Sergey/Medium/index.html]: Similarly.

File: Sergey/Medium/index.html [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+                    </div>
+                    <div class="item">
+                        <div class="fuck">
```
Comment (iruseg00) [inline Sergey/Medium/index.html]: Similarly.

File: Sergey/Medium/index.html [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+                            <i class="far fa-circle"></i>
+                            <h4>Maya Kosoff</h4>
+                        </div>
+                        <h2>A Tale Of Two Kinds of Media Workers</h2>
+                        <div class="date">
+                            <span>Oct 20</span>
+                            <span> | </span>
+                            <span>3 min read</span>
+                        </div>
```
Comment (iruseg00) [inline Sergey/Medium/index.html:93]: Similarly.

File: Sergey/Medium/index.html [hunk 10]
```diff
@@ -0,0 +1,188 @@
...
+                    </div>
+                </div>
+                <div class="items_number">
+                    <div class="number">
+                        <span>04</span>
```
Comment (iruseg00) [inline Sergey/Medium/index.html]: Similarly.

File: Sergey/Medium/index.html [hunk 11]
```diff
@@ -0,0 +1,188 @@
...
+                    </div>
+                    <div class="item">
+                        <div class="fuck">
```
Comment (iruseg00) [inline Sergey/Medium/index.html]: Similarly.

File: Sergey/Medium/index.html [hunk 12]
```diff
@@ -0,0 +1,188 @@
...
+                            <i class="far fa-circle"></i>
+                            <h4>Robert Roy Britt in Elemental</h4>
+                        </div>
+                        <h2>10 Signs the Pandemic Is About to Get Much Worse</h2>
+                        <div class="date">
+                            <span>Oct 19</span>
+                            <span> | </span>
+                            <span>8 min read</span>
+                        </div>
```
Comment (iruseg00) [inline Sergey/Medium/index.html:108]: Similarly.

File: Sergey/Medium/index.html [hunk 13]
```diff
@@ -0,0 +1,188 @@
...
+                    </div>
+                </div>
+                <div class="items_number">
+                    <div class="number">
+                        <span>05</span>
+                    </div>
```
Comment (iruseg00) [inline Sergey/Medium/index.html:114]: Similarly.

File: Sergey/Medium/index.html [hunk 14]
```diff
@@ -0,0 +1,188 @@
...
+                    <div class="item">
+                        <div class="fuck">
```
Comment (iruseg00) [inline Sergey/Medium/index.html]: Similarly.

File: Sergey/Medium/index.html [hunk 15]
```diff
@@ -0,0 +1,188 @@
...
+                            <i class="far fa-circle"></i>
+                            <h4>Tristan Joshua Alba in Towards Date Science</h4>
+                        </div>
+                        <h2>5 Reasons why I'm learning Web Development, as a Date Scientist</h2>
+                        <div class="date">
+                            <span>Oct 20</span>
+                            <span> | </span>
+                            <span>8 min read</span>
```
Comment (iruseg00) [inline Sergey/Medium/index.html]: Similarly.

File: Sergey/Medium/index.html [hunk 16]
```diff
@@ -0,0 +1,188 @@
...
+                        </div>
+                    </div>
+                </div>
+                <div class="items_number">
+                    <div class="number">
+                        <span>06</span>
```
Comment (iruseg00) [inline Sergey/Medium/index.html]: Similarly.

File: Sergey/Medium/index.html [hunk 17]
```diff
@@ -0,0 +1,188 @@
...
+                    </div>
+                    <div class="item">
+                        <div class="fuck">
```
Comment (iruseg00) [inline Sergey/Medium/index.html]: Similarly.

File: Sergey/Medium/index.html [hunk 18]
```diff
@@ -0,0 +1,188 @@
...
+                            <i class="far fa-circle"></i>
+                            <h4>Kat Rosenfield  in Arc Digital</h4>
+                        </div>
+                        <h2>White Woman Need Not Apply</h2>
+                        <div class="date">
+                            <span>Oct 19</span>
+                            <span> | </span>
+                            <span>5 min read</span>
+                        </div>
```
Comment (iruseg00) [inline Sergey/Medium/index.html:138]: Similarly.

File: Sergey/Medium/index.html [hunk 19]
```diff
@@ -0,0 +1,188 @@
...
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class="tegs">
```
Comment (iruseg00) [inline Sergey/Medium/index.html]: The word "tag" is written with "a"))

File: Sergey/Medium/index.html [hunk 20]
```diff
@@ -0,0 +1,188 @@
...
+        <div class="tegs_conteiner">
```
Comment (iruseg00) [inline Sergey/Medium/index.html]: "Container" is written with "a") But you named the next container successfully)

File: Sergey/Medium/index.html [hunk 21]
```diff
@@ -0,0 +1,188 @@
...
+            <div class="tegs_container1">
+                <p>1</p>
+            </div>
```
Comment (iruseg00) [inline Sergey/Medium/index.html:148]: Why it is container if he consist one element?

File: Sergey/Medium/index.html [hunk 22]
```diff
@@ -0,0 +1,188 @@
...
+            <div class="tegs_container2">
+                <div class="tegs_container2_top">
```
Comment (iruseg00) [inline Sergey/Medium/index.html]: Very often the word "container" is repeated. Try to describe what is happening in the container with the name, less numbers.

File: Sergey/Medium/style.css [hunk 23]
```diff
@@ -0,0 +1,430 @@
...
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+body
+{
+    text-rendering: optimizeLegibility;
+    height: 2000px;
+    min-height: 100vh;
```
Comment (iruseg00) [inline Sergey/Medium/style.css]: Choose one of two: height or min-height.

File: Sergey/Medium/index.html (hunk 2 above)
Comment (Green1ce) [reply Sergey/Medium/index.html]: > Why is span here?
Why not span? Line element

File: Sergey/Medium/index.html (hunk 3 above)
Comment (Green1ce) [reply Sergey/Medium/index.html]: > The class name should describe the essence of the element.
The bottom line is that I spent a lot of time until I did the right thing and my soul suggested what to call it)

File: Sergey/Medium/index.html (hunk 4 above)
Comment (Green1ce) [reply Sergey/Medium/index.html:63]: > Can we remove the 3 span and leave the text in the one div?
Yes, but in the example site, these elements, as I understand it, automatically set the date and time of reading (configured separately)

File: Sergey/Medium/index.html (hunk 20 above)
Comment (Green1ce) [reply Sergey/Medium/index.html]: > "Container" is written with "a") But you named the next container successfully)
It happens)

File: Sergey/Medium/index.html (hunk 19 above)
Comment (Green1ce) [reply Sergey/Medium/index.html]: > The word "tag" is written with "a"))
Google didn't correct my)

File: Sergey/Medium/index.html (hunk 21 above)
Comment (Green1ce) [reply Sergey/Medium/index.html:148]: > Why it is container if he consist one element?The task was to make the right side of the site - for the left, I just made a container and left

File: Sergey/Medium/index.html (hunk 22 above)
Comment (Green1ce) [reply Sergey/Medium/index.html]: > Very often the word "container" is repeated. Try to describe what is happening in the container with the name, less numbers.
Nothing else came to mide(

File: Sergey/Medium/style.css (hunk 23 above)
Comment (Green1ce) [reply Sergey/Medium/style.css]: > Choose one of two: height or min-height.
It's must be width =\

File: Sergey/Medium/index.html (hunk 2 above)
Comment (iruseg00) [reply Sergey/Medium/index.html]: Extra nesting.

File: Sergey/Medium/index.html (hunk 3 above)
Comment (iruseg00) [reply Sergey/Medium/index.html]: Fairly))

File: Sergey/Medium/index.html (hunk 4 above)
Comment (iruseg00) [reply Sergey/Medium/index.html:63]: May be. But on now better delete extra nesting.

File: Sergey/Medium/index.html [hunk 24]
```diff
@@ -48,137 +50,125 @@ <h1 class="article_header">Where good ideas find you</h1>
             <div class="items_items">
                 <div class="items_number">
                     <div class="number">
-                        <span>01</span>
+                        <p>01</p>
```
Comment (iruseg00) [inline Sergey/Medium/index.html:53]: do this
do thsis

File: Sergey/Medium/index.html [hunk 25]
```diff
@@ -48,137 +50,125 @@ <h1 class="article_header">Where good ideas find you</h1>
...
-                            <span> | </span>
-                            <span>7 min read</span>
+                            Oct 21 | 7 min read
                         </div>
                     </div>
                 </div>
                 <div class="items_number">
                     <div class="number">
-                        <span>02</span>
+                        <p>02</p>
```
Comment (iruseg00) [inline Sergey/Medium/index.html:68]: Similarly.

File: Sergey/Medium/index.html [hunk 26]
```diff
@@ -48,137 +50,125 @@ <h1 class="article_header">Where good ideas find you</h1>
...
-                            <span> | </span>
-                            <span>3 min read</span>
+                            Oct 21 | 3 min read
                         </div>
                     </div>
                 </div>
                 <div class="items_number">
                     <div class="number">
-                        <span>03</span>
+                        <p>03</p>
```
Comment (iruseg00) [inline Sergey/Medium/index.html:83]: Similarly.

File: Sergey/Medium/index.html [hunk 27]
```diff
@@ -48,137 +50,125 @@ <h1 class="article_header">Where good ideas find you</h1>
...
-                            <span> | </span>
-                            <span>3 min read</span>
+                            Oct 20 | 3 min read
                         </div>
                     </div>
                 </div>
                 <div class="items_number">
                     <div class="number">
-                        <span>04</span>
+                        <p>04</p>
```
Comment (iruseg00) [inline Sergey/Medium/index.html:98]: Similarly.

File: Sergey/Medium/index.html [hunk 28]
```diff
@@ -48,137 +50,125 @@ <h1 class="article_header">Where good ideas find you</h1>
...
-                            <span> | </span>
-                            <span>8 min read</span>
+                            Oct 19 | 8 min read
                         </div>
                     </div>
                 </div>
                 <div class="items_number">
                     <div class="number">
-                        <span>05</span>
+                        <p>05</p>
```
Comment (iruseg00) [inline Sergey/Medium/index.html:113]: Similarly.

File: Sergey/Medium/index.html [hunk 29]
```diff
@@ -48,137 +50,125 @@ <h1 class="article_header">Where good ideas find you</h1>
...
-                            <span> | </span>
-                            <span>8 min read</span>
+                            Oct 20 | 8 min read
                         </div>
                     </div>
                 </div>
                 <div class="items_number">
                     <div class="number">
-                        <span>06</span>
+                        <p>06</p>
```
Comment (iruseg00) [inline Sergey/Medium/index.html:128]: Similarly.

File: Sergey/Medium/index.html (hunk 24 above)
Comment (Green1ce) [reply Sergey/Medium/index.html:53]: > do this
> do thsis
Realy. lol)

**Pull request merged**